docs/developing/publishing-apps.md

```diff
@@ -0,0 +1,105 @@
+# App publishing guide
+
+After you've completed developing and testing your app's package for Sandstorm, you can publish it in the [Sandstorm App Market](https://apps.sandstorm.io/).
+
+## Verify your identity
+
+### Sign up with [Keybase.io](https://keybase.io)
+
+Currently, Keybase is invite-only. If you need an invite, you can contact [Jade](mailto:jade@sandstorm.io) or [Kenton](mailto:kenton@sandstorm.io). You should connect some of your public identites with your Keybase account, like Twitter and Github.
+
+Sandstorm app authors are verified using a PGP key linked to Keybase. You should get the [prerequisites](https://keybase.io/docs/command_line/prerequisites) and follow [their directions](https://keybase.io/docs/command_line/installation) to get their software set up.
+
+### Sign your app
+
+In order to verify that you are the author of the app in question, you need to sign the following ASCII statement: `I am the author of the Sandstorm.io app with the following ID: <app-id>`, where `<app-id>` is the one from your `sandstorm-pkgdef.capnp` file.
+
+To generate a pgp-signature file using gpg, run a command like this:
+
+`echo -n "I am the author of the Sandstorm.io app with the following ID: <app-id>" |
+     gpg --sign > pgp-signature`
```

Comment:
You had asked how to do this on OS X & Windows, via vagrant-spk.

For now, the answer has to be that you do the signing on OS X & Windows, following Keybase's instructions (presumably) for installing GPG on OS X or Windows.

We should update `vagrant-spk` to incorporate more direct GPG support, to avoid this problem. Would you be willing to file an issue against vagrant-spk here, about that? https://github.com/sandstorm-io/vagrant-spk/issues

The simplest thing we can do probably is to tell people to run:

```
echo -n "I am the author of the Sandstorm.io app with the following ID: <app-id>" | vagrant-spk ssh -c gpg --sign > pgp-signature
```

but we'd need to test that and particularly make sure that people's GPG keyring flows into the Vagrant box, and/or that we show people how to share their GPG keyring with the Vagrant box.

I don't know if that's actually easier than installing GPG on OS X or Windows.